chrisarcand/miqversions
# Use rust

I do it all for the :trollface:


Description
-----------

This makes a language switch to use Rust for teh lolz.  Mostly did it because it is a easier and (in my humble opinion) more sane developer workflow experience.

Note:  This probably means updating `chrisarcand/formulae`, but that is even if I can get the approval of the `miqversions` Czar himself, Mr. @chrisarcand, to even approve this change, so we will cross that bridge when we come to it.


QA
--

To run, just install `rust` in whatever form you see fit, and then build the project:

```console
$ cargo build --release
$ target/release/miqversions
```
